### ISSUE TITLE
[DOCS] Create a GitHub PR Template

### ISSUE THREAD

**Issue by rubemfsv** [rubemfsv/Github-Explorer-Mobile-Clean-Architecture#6]
**Description**

Please create a GitHub PR (Pull Request) template to streamline the process of submitting and reviewing pull requests in our repository.

**Expected Behavior**

The PR template should serve as a standardized guide for contributors to provide essential information when submitting pull requests. It should include sections for a clear title, a detailed description of the changes, and any relevant context.

**Actual Behavior**

Currently, we don't have a PR template in place, which can lead to inconsistent or incomplete information being provided with PR submissions. Having a template will improve the PR submission process and make it easier for reviewers to understand and assess the changes.

**Additional Information**

Adding a PR template will contribute to better organization and communication within our project. It will help both contributors and reviewers by ensuring that necessary information is included with each PR, streamlining the review process, and reducing back-and-forth discussions. Please create the template based on common PR guidelines and best practices.

Follow the [pact-js](https://github.com/pact-foundation/pact-js/) PR template pattern, but adapting to Clean React App context. Don't forget to ensure commit messages are following the conventional commits.

**Comment by ghOst-vedant** [comment]
hey @rubemfsv. this template need to be submitted here in the comments??

**Comment by rubemfsv** [comment]
@ghOst-vedant no, it should be in the .github, as this example in my other project: https://github.com/rubemfsv/clean-react-app

**Comment by ghOst-vedant** [comment]
@rubemfsv i'll finish by today evening

**Comment by ghOst-vedant** [comment]
@rubemfsv you available?? is this fine






# Pull Request

## Description



## Type of Change


- [ ] Bug Fix
- [ ] New Feature
- [ ] Enhancement / Optimization
- [ ] Documentation Update
- [ ] Code Refactoring
- [ ] Other (please specify):


# Commit messages

Our changelog is automatically built from our commit history, using conventional changelog. This means we'd like to take care that:

- commit messages with the prefix `fix:` or `fix(foo):` are suitable to be added to the changelog under "Fixes and improvements"
- commit messages with the prefix `feat:` or `feat(foo):` are suitable to be added to the changelog under "New features"

If you've made many commits that don't adhere to this style, we recommend squashing 
your commits to a new branch before making a PR. Alternatively, we can do a squash
merge, but you'll lose attribution for your change.

## Checklist



- [ ] Code follows Clean React App style and conventions.
- [ ] Code adheres to Clean React App folder structure.
- [ ] Code passes existing tests.
- [ ] New tests added to cover changes (if applicable).
- [ ] Documentation updated (if applicable).
- [ ] Commits follow conventional commits format.

## Screenshots / GIFs (if applicable)



## Additional Context

**Comment by rubemfsv** [comment]
@ghOst-vedant sorry for my late reply. Yes! Its ok
